src/e3/cve.py

```diff
@@ -94,7 +101,7 @@ def search_by_cpe_name(
 
         start_index = 0
         while True:
-            r = self.session.get(url + f"&startIndex={start_index}", headers=headers)
+            r = self._session.get(url + f"&startIndex={start_index}", headers=headers)
```

Comment:
To me we should call for `self.session` here.

Reply:
I didn't say anything, it doesn't make sense, or would force me to have the same warning twice :)